### DISCUSSION TITLE
你的eSearch网站里下载链接是错误的

### DISCUSSION THREAD

**Discussion by q195com** [xushengfeng/eSearch#266] [General]
![屏幕截图 2024-08-30 135745](https://github.com/user-attachments/assets/43e78b4e-c21b-4ccb-9b91-03b95b2749c9)
点击这个图片的下载链接如下：
eSearch网站下载链接：https://git.xfj0.cn/https://github.com/xushengfeng/eSearch/releases/download/1.12.4/eSearch-1.12.4-win32-x64.exe
实际下载链接：https://github.com/xushengfeng/eSearch/releases/download/1.12.4/eSearch-1.12.4-win32-x64.exe

**Comment by xushengfeng** [comment]
这是使用了加速链接后的连接
不过这个加速镜像可能被屏蔽了
如果使用加速镜像，刷新几次网站，否则，不勾选加速链接

**Reply by q195com** [reply]
好的，收到
